### PR TITLE
JDG-2280 main readme

### PR DESCRIPTION
**Pull request by oraNod**

### CODE REVIEW

File: openshift/start-minishift.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -e
 
 
 startMinishift() {
-  minishift start
+  minishift start --ocp-tag=latest
```
Comment (galderz) [inline openshift/start-minishift.sh]: Hmmmm, why did you add this? This could make the whole experience a moving target. If I install a minishift version, say for openshift 3.10, when 3.11 comes out it'll try that and keep moving forward. Things might break... I'd leave up to the user to make a conscious decision on which Minishift version to use.

File: openshift/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+1. Download and install minishift.  
+>The [Red Hat Container Development Kit](https://developers.redhat.com/products/cdk/overview/) includes minishift.
+
+2. Clone this repository to your host.
+3. Set your Red Hat customer account credentials with the `REDHAT_REGISTRY_USER` and `REDHAT_REGISTRY_PASSWORD` environment variables.
+```bash
+$ export REDHAT_REGISTRY_USER=username@redhat.com
+$ export REDHAT_REGISTRY_PASSWORD=password
+```
+4. Run the following script to configure minishift:
```
Comment (galderz) [inline openshift/README.md]: It'd be stress that this normally only requires to be executed once.

**Review by galderz — COMMENTED** [review]
Some minor comments :)

File: openshift/start-minishift.sh (hunk 1 above)
Comment (oraNod) [reply openshift/start-minishift.sh]: hey, @galderz I knew adding this option would be debatable and can revert it. the reason I added it though is that - in docs - we advise users to go with the latest version of OpenShift. also, according to PM, 3.11 is the minimum version for 7.3. if you think it's better, we can keep it as part of the requirements in the readme and let users make the conscious decision as you said.

File: openshift/setup-minishift.sh [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 set -e -x
 
-minishift profile set openshift-datagrid-tutorials
+minishift profile set datagrid-quickstart
```
Comment (oraNod) [inline openshift/setup-minishift.sh:5]: @galderz I hope this name change is ok. `datagrid-quickstart` seems like an easier profile name to work with.

File: openshift/start-minishift.sh (hunk 1 above)
Comment (oraNod) [reply openshift/start-minishift.sh]: also, I found that there is a config option for this: `openshift-version` so it would be better to include in the setup script as `minishift config set openshift-version`, not that it matters. I've removed that option anyway. cheers.

File: openshift/README.md (hunk 2 above)
Comment (galderz) [reply openshift/README.md]: @oraNod Did you address this comment?

File: openshift/start-minishift.sh (hunk 1 above)
Comment (galderz) [reply openshift/start-minishift.sh]: Ok, let's leave it out for now.

File: openshift/README.md (hunk 2 above)
Comment (oraNod) [reply openshift/README.md]: @galderz yes, please see https://github.com/jboss-developer/jboss-jdg-quickstarts/pull/223/files#diff-b0bc969807239fee0eb31a9dbb4238fbR30
